static/specification.html

```diff
@@ -301,24 +302,24 @@ <h2>
         </dt>
         <dd>
           The party receiving payments on behalf of the website, whose details
-          are provided by a [=payment end-point=].
+          are provided by a [=payment pointer=].
         </dd>
         <dt>
-          <dfn>Payment end-point</dfn>:
+          <dfn>Payment Pointer</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a <a data-cite="Open Payments#specification">Open
-          Payments account end-point</a> (i.e., a JSON resource containing
+          A [=URL=] to an <a data-cite="Open Payments#specification">Open
+          Payments API entry-point</a> (i.e., a JSON resource containing
           details that facilitate payment to an account).
         </dd>
         <dt>
           <dfn>Payment session</dfn>
         </dt>
         <dd>
           An session between a [=monetization provider=] and [=monetization
-          receiver=] initiated by the creation of an incoming payment resource
-          at the [=monetization receiver=]. One or more payments can be sent by
-          the [=monetization provider=] in a single session.
+          receiver=] initiated by the [=monetization provider=] at the
+          [=monetization receiver=]. One or more payments can be sent by the
+          [=monetization provider=] in a single session.
```

Comment:
```suggestion
          [=monetization receiver=]. One or more payments can be initiated by the
          [=monetization provider=] in a single session.
```
The monetization provider could send it out of their account or out of the linked wallet's account.